threedi_api_qgis_client/workers.py

```diff
@@ -34,19 +34,19 @@ def __init__(self, api_client, wss_url, model_id=None):
         self.simulations_list = []
         self.model_id = model_id
 
-
     @pyqtSlot()
     def run(self):
         """Checking running simulations progresses."""
         try:
             self.tc = ThreediCalls(self.api_client)
             if self.model_id:
-                logger.debug("Fetching simulations list and filtering it on model id %s", self.model_id)
+                logger.debug(f"Fetching simulations list and filtering it on model id {self.model_id}")
```

Comment:
For logging the old `%s` way, in combination with the comma, is preferred. This way, error reporting tools like sentry can group similar entries, which it can't if you pre-format the error string. Now.... we don't use sentry in qgis so it doesn't really matter here, but preferably don't change the "style" that we try to use in the web side of the code. Unless it is really weird compared to the rest of the surrounding code.
See https://reinout.vanrees.org/weblog/2015/06/05/logging-formatting.html for the technical reason. (The blog post predates the admittedly nice f-strings).

A couple of lines below you changed the `"error %s message" % something` style: fine to change that, as it isn't the style we prefer anyway :-)

Reply:
It would be nice to keep consistent strings formatting style at least on single module level, so I would like to stick with f-strings as long as we don't use `sentry` within plugin. If there will be such need - we can make a change in more consisten way and not mix strings formatting styles where it's not necessary.